backend/app/services/attachment/attachment_service.py

```
@@ -98,7 +101,28 @@ def upload_attachment(
+                user_id=user_id,
+                attachment_id=attachment.id,
+                original_filename=filename,
+            )
+            minio_client.upload_file(
+                object_key=object_key,
+                data=binary_data,
+                content_type=mime_type,
+            )
+            attachment.minio_object_key = object_key
+            logger.info(f"Uploaded attachment {attachment.id} to MinIO: {object_key}")
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)